roles/ood_matlab/tasks/main.yaml

```diff
@@ -8,7 +8,7 @@
     dest: /var/www/ood/apps/sys/matlab/
     clone: yes
 
-name: Replace submit.yml.erb
+- name: Replace submit.yml.erb
```

Comment:
Just a note, avoid adding unrelated changes to a commit.  The summary for the commit is about the httpd task so the changes should only relate to that task.  Make a correction like a missing dash a separate commit.  This makes it easier to track changes and cherry pick if needed.